CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 - Allow to pass Cloud authentication token via TUIST_CLOUD_TOKEN even when not CI [#3380](https://github.com/tuist/tuist/pull/3380) by [@danyf90](https://github.com/danyf90)
 
+
+### Changed
+
+- **Breaking** Minimum supported Xcode version bumped to 12.4.
```

Comment:
Not related to this PR, but could you please merge the 2 `Fixed` section? 🙏

Reply:
done